src/main/java/mcp/mobius/waila/client/ProxyClient.java

```diff
@@ -68,7 +68,9 @@ public static class WorldUnloadEventHandler {
 
         @SubscribeEvent
         public void onWorldUnload(WorldEvent.Unload event) {
-            DataAccessorCommon.instance = new DataAccessorCommon();
+            if (event.world.isRemote) {
+                DataAccessorCommon.instance = new DataAccessorCommon();
```

Comment:
Just for understanding, why is this needed?

Reply:
Waila is a client mod, it makes no sense to update stuff when the server unloads a world